guide/src/credentials/override_credentials.md

```diff
@@ -0,0 +1 @@
+# Override the authentication credentials
```

Comment:
What is this file?

Reply:
I was moving things around and `mdbook` kept re-creating some files with previous names. I'll clean up those extra files.